### PR TITLE
Add ClaimCreatedEvent and ClaimModifiedEvent

### PR DESCRIPTION
**Pull request by Narimm**
1. Claim creation event... this is fired once all the claim creation checks have been passed.  It can be cancelled. It contains the claim and the creating player
2. Claim modification event - this is fired as the claim is modified - it cannot be cancelled. It contains the claim and the modifier...which can be null.

Additional changes
- The datastores basic claim create event is overloaded to allow us to differentiate between real new claims and modification of claims.
- Cleaned up the datastore imports..

Use Case:
This could be used so other plugins can hook gp and perform an action if a claim is created or changed. Something as simple as logging claim creations/modifications and deletions.

### CODE REVIEW

File: src/main/java/me/ryanhamshire/GriefPrevention/DataStore.java [hunk 1]
```diff
@@ -19,14 +19,10 @@
...
 import me.ryanhamshire.GriefPrevention.events.ClaimDeletedEvent;
-import org.bukkit.Bukkit;
-import org.bukkit.ChatColor;
-import org.bukkit.Chunk;
-import org.bukkit.Location;
-import org.bukkit.Material;
-import org.bukkit.OfflinePlayer;
-import org.bukkit.World;
+import me.ryanhamshire.GriefPrevention.events.ClaimModifyEvent;
+import org.bukkit.*;
```
Comment (RoboMWM) [inline src/main/java/me/ryanhamshire/GriefPrevention/DataStore.java]: Can you make your IDE not do this please (wildcard imports)? Thanks.

**Review by RoboMWM — DISMISSED** [review]
Looks good, will pull after the 1.13 PR.

**Comment by RoboMWM** [comment]
Only other minor critique is that the event name could be swapped to something more consistent such as ModifyClaimEvent.

**Comment by Narimm** [comment]
I adjusted the Event name ...but to keep it consistent with existing events I went with ClaimModifiedEvent...
preexisting ClaimDeleteEvent...

**Comment by Narimm** [comment]
So the new ones are 
1. ClaimCreatedEvent
2.ClaimModifiedEvent
(Existing) 
ClaimDeletedEvent

**Review by RoboMWM — APPROVED** [review]

**Pull request merged**